middleware/auth.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require('jsonwebtoken');
+const { super_secret } = require('../utils/secrets');
+
+const jwtKey = process.env.JWT_SECRET || super_secret;
+
+module.exports = (req, res, next) => {
+  const token = req.headers.authorization;
+
+  if (token) {
+    jwt.verify(token, jwtKey, (err, decoded) => {
+      if (err) {
+        console.error(err);
+        res.status(401).json({ message: 'Invalid Credentials' });
```

Comment:
Can we add these errors to the util and remove hardcoding in here?

Reply:
Will do, after your PR is merged, so I don't cause any merge conflicts.